torch/csrc/jit/script/compiler.cpp

```diff
@@ -1909,7 +1909,7 @@ struct to_ir {
           type,
           emitExpr(apply.inputs()[1], type),
           /*allow_conversions=*/true);
-      if (!expr->type()->isSubtypeOf(type)) {
+      if (!type->isSubtypeOf(expr->type())) {
```

Comment:
Huh? This flips the subtyping from `actual <: formal` to `formal <: actual`,  which is not right.

Reply:
nice catch!